e2e-tests/tests/infra/StudioPlaywrightPage.ts

```diff
@@ -233,4 +233,32 @@ export default class StudioPlaywrightPage {
   getComponentPath(componentName: string) {
     return path.join(this.tmpDir, "src/components", componentName + ".tsx");
   }
+
+  async waitForIFrameImagesToLoad() {
+    const images = await this.preview.getByRole("img").all();
+    const imgPromises = images.map((img) =>
+      expect
+        .poll(
+          async () => {
```

Comment:
I'm pretty sure you don't need the async + await here, it can just be `.poll(() => img.evaluate(...)`

Reply:
I tried that and it works without the async/await, but I wasn't entirely sure about how the mechanics of that worked.  I thought that `expect.poll` keeps polling the function provided, and only passes when (in this case) the function returns `true`.  But `image.evauate` returns a boolean promise, so I wasn't sure how exactly it was working.  Does it already know to wait for the promise to resolve?

Reply:
Oh hmmm it looks like the callback can return a promise?
![image](https://github.com/yext/studio/assets/23005393/34fa160e-2554-46c3-a295-3115ca6a99ad)
I just assumed it would work because 
`async () => { return await Promise.resolve() }` is equivalent to `async () => { return Promise.resolve }`

Reply:
so if your old code was working this would have to work too

Reply:
ahh interesting, gotcha